### PR TITLE
Issue #15214: Migrated section 4.4 Column Limit: 100 to whole config testing

### PR DESCRIPTION
**Pull request by Zopsss**
#15214

### CODE REVIEW

File: src/it/resources/com/google/checkstyle/test/chapter4formatting/rule44columnlimit/InputColumnLimit.java [hunk 1]
```diff
@@ -1,203 +1,40 @@
 package com.google.checkstyle.test.chapter4formatting.rule44columnlimit; // ok
 
-import com.google.checkstyle.test.chapter3filestructure.toolongpackagetotestcoveragegooglesjavastylerule.*; // ok
+import com.google.checkstyle.test.chapter3filestructure.toolongpackagetotestcoveragegooglesjavastylerule.PackageStatementTest; // ok
```
Comment (Zopsss) [inline src/it/resources/com/google/checkstyle/test/chapter4formatting/rule44columnlimit/InputColumnLimit.java:3]: had to change import cuz of `no "*" imports` violation

File: src/it/resources/com/google/checkstyle/test/chapter4formatting/rule44columnlimit/InputColumnLimit.java [hunk 2]
```diff
@@ -1,203 +1,39 @@
...
-  private static int sNumCreated = 0;
-
-  /** Invalid format * */
-  private int badMember = 2;
-
-  /** Valid format * */
-  private int mNumCreated1 = 0;
-
-  /** Valid format * */
-  protected int mNumCreated2 = 0;
```
Comment (Zopsss) [inline src/it/resources/com/google/checkstyle/test/chapter4formatting/rule44columnlimit/InputColumnLimit.java:33]: deleted unnecessary code which was not related to this section

File: src/it/resources/com/google/checkstyle/test/chapter4formatting/rule44columnlimit/InputColumnLimit.java [hunk 3]
```diff
@@ -1,203 +1,39 @@
...
+   *
    * @param badFormat1 bad format
    * @param badFormat2 bad format
    * @param badFormat3 bad format
+   * @return hack
    * @throws java.lang.Exception abc
    */
   int test1(int badFormat1, int badFormat2, final int badFormat3) throws java.lang.Exception {
     return 0;
   }
```
Comment (Zopsss) [inline src/it/resources/com/google/checkstyle/test/chapter4formatting/rule44columnlimit/InputColumnLimit.java:27]: kept this method on purpose to show that violation is given only on code which is longer than column limit 100

**Review by Zopsss — COMMENTED** [review]
item:

File: src/it/resources/com/google/checkstyle/test/chapter4formatting/rule44columnlimit/InputColumnLimit.java (hunk 3 above)
Comment (Zopsss) [reply src/it/resources/com/google/checkstyle/test/chapter4formatting/rule44columnlimit/InputColumnLimit.java:27]: had to re-arrange block tags to make them in proper order

**Review by romani — APPROVED** [review]
ok to merge

**Review by rdiachenko — APPROVED** [review]
lgtm

**Pull request merged**